mrp_product_variants/README.rst

```diff
@@ -28,6 +28,11 @@ In case of trouble, please check there if your issue has already been reported.
 If you spotted it first, help us smashing it by providing a detailed and welcomed feedback
 `here <https://github.com/odoomrp/odoomrp-wip/issues/new?body=module:%20mrp_product_variants%0Aversion:%208.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
 
+Known issues / Roadmap
+======================
+
+You can add variant attributes from the same type in the BoM list, if at least
```

Comment:
This is not an issue, because it's the intended behaviour. You should include it the usage section (and you can expand a bit this section with some explanation).